schemas/answers/checkbox.json

```diff
@@ -13,6 +13,9 @@
       "label": {
         "$ref": "https://eq.ons.gov.uk/common_definitions.json#/non_empty_string"
       },
+      "no_label": {
+        "type": "boolean"
+      },
```

Comment:
If the only way we are going to solve this for now is with a new property then I think it would be better to have something like `show_label` which can be default to `True` in runner. The `"no_label": false` double negative isn't that great. Might be worth discussing with the rest of the team, have discussed with Lewis and I believe he is on board. I don't think there is a perfect solution but think "show" is clearer.

Reply:
Added `show_label` now set to `False` if label shouldn't be visible. Thanks!